modular_skyrat/modules/morgue_radio/morgue.dm

```diff
@@ -0,0 +1,41 @@
+/obj/structure/bodycontainer/morgue/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/morgue_radio)
+
+/obj/structure/bodycontainer
```

Comment:
Should be on morgues, and not all bodycontainers, I feel like.
```suggestion
/obj/structure/bodycontainer/morgue
```

Reply:
Oh yeah good catch, since I believe crematoriums will just incinerate the radio anyway.